ai-platform-unified/notebooks/unofficial/hyperparameter_tuning/distributed-hyperparameter-tuning.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.      parser.add_argument(
</pre><p>Convention is "units" not "num_neurons"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Done